bin/install

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+brew install --devel go
+brew tap homebrew/science
+brew install vips
+go get github.com/tools/godepgodep
+go get github.com/opendoor-labs/gothumb
+cd $GOPATH/src/github.com/opendoor-labs/gothumb/
```

Comment:
non-Go users won't have a $GOPATH set up, and they'd need to do so before running any of the `go` commands in this script